src/neuroglancer/sliceview/backend_chunk_decoders/bossNpz.ts

```diff
@@ -15,7 +15,7 @@
  */
 
```

Comment:
There isn't really a need to rename this file.

Reply:
the spec is slightly different between ndstore and boss with the file duplication occurring on the initial BOSS feature add.  sSems easier at this point to just delete the ndstore file?

Reply:
Oh, good point.  I didn't realize that.  The Python backend supports multiple channels and therefore needs the ndstore version, though, and they are trivial enough that there may not be a point in trying to combine them.  Perhaps just leave both versions.